src/tools/rbd/Utils.cc

```diff
@@ -446,6 +446,37 @@ int get_pool_image_snapshot_names(const po::variables_map &vm,
   return 0;
 }
 
+int get_pool_snapshot_names(const po::variables_map &vm,
```

Comment:
Nit: probably a good idea to ensure that ```--[<mod>-]image``` hasn't been populated when expecting to open by id